src/libs/formatDateToString.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Date 형태로 들어오는 값을 yyyy.mm.dd 형태의 문자열로 변환해주는 함수입니다.
+ * @param date - Date 타입의 값을 넣어주세요.
+ * @returns yyyy.mm.dd 형태의 문자열
+ */
+
+export const formatDateToString = (date: Date) => {
+  const year = date.getFullYear();
+  const month = `${date.getMonth() < 9 ? '0' : ''}${date.getMonth() + 1}`;
+  const day = `${date.getDate() < 9 ? '0' : ''}${date.getDate()}`;
```

Comment:
padStart()메소드로 좀 더 쉽게 구현할 수 있을 것 같네요.(month값 조건문 넣을 때 1 더한 걸로 해야하지 않나요?)
```suggestion
  const month = (date.getMonth() + 1).toString().padStart(2, '0')
  const day = date.getDate().toString().padStart(2, '0');
```

Reply:
어머머머 그러네요!!! 언넝 수정하겠습니다! 오 padStart 메서드는 첨봐요!! 증멜 아는 메서드가 많으시군용,, 멤모멤모 ✍️